taca/utils/bioinfo_tab.py

```diff
@@ -51,30 +58,40 @@ def collect_runs():
              if rundir_re.match(os.path.basename(os.path.abspath(run_dir))) and os.path.isdir(run_dir):
                 #update the run status
                 update_statusdb(run_dir)
+    
 
-
-
+# Updaterun command
+# Gets status for a project
 def update_statusdb(run_dir):
-    project_ids=get_ss_projects(run_dir)
+    #fetch individual fields
+    project_info=get_ss_projects(run_dir)
     run_name = os.path.basename(os.path.abspath(run_dir))
-    status=get_status(run_dir)
+    
     couch=setupServer(CONFIG)
     valueskey=datetime.datetime.now().isoformat()
     db=couch['bioinfo_analysis']
     view = db.view('full_doc/pj_run_to_doc')
-    for p in project_ids:
-        obj={'run_id':run_name, 'project_id':p, 'status':status, 'values':{valueskey:{'user':'taca','status':status}} }
-        if len(view[[p, run_name]].rows) == 1:
-            remote_doc= view[[p, run_name]].rows[0].value
-            remote_status=remote_doc["status"]
-            if remote_status in ['Incoming', 'Sequencing Done', 'Demultiplexing', 'Demultiplexed', 'Transferring']:
-                final_obj=merge(obj, remote_doc)
-                logger.info("saving {} {} as  {}".format(run_name, p, status))
-                db.save(final_obj)
-        else:
-            logger.info("saving {} {} as  {}".format(run_name, p, status))
-            db.save(obj)
+    #Construction and sending of individual records
+    for p in project_info:
+        for flowcell in project_info[p]:
+            for lane in project_info[p][flowcell]:
+                for sample in project_info[p][flowcell][lane]:
+                    sample_status = project_info[p][flowcell][lane][sample].value
+                    
+                    obj={'run_id':run_name, 'project_id':p, 'flowcell': flowcell, 'lane': lane, 'sample':sample,
+                        'values':{valueskey:{'user':'taca','sample_status':sample_status}} }
+                    if len(view[[p, flowcell, lane, sample]].rows) == 1:
+                        remote_doc= view[[p, flowcell, lane, sample]].rows[0].value
+                        remote_status=remote_doc["sample_status"]
+                        if remote_status in ['Incoming', 'Sequencing Done', 'Demultiplexing', 'Demultiplexed', 'Transferring']:
+                            final_obj=merge(obj, remote_doc)
+                            logger.info("saving {} {} as  {}".format(run_name, p, sample_status))
+                            db.save(final_obj)
+                    else:
+                        logger.info("saving {} {} as  {}".format(run_name, p, sample_status))
```

Comment:
this logging needs to be updated with the new granularity level, also the one in line 88